src/core/colorspace.js

```diff
@@ -294,13 +294,10 @@ var ColorSpace = (function ColorSpaceClosure() {
     }
 
     cs = xref.fetchIfRef(cs);
-    var mode;
```

Comment:
Fixed in the new commit. Now that you mentioned this, I also noticed that it's not used outside of this method, so it may just as well be only local.